### ISSUE TITLE
Provide Scripts for Downloading Internal and External Data

### ISSUE THREAD

**Issue by wjbmattingly** [project-lux/data-pipeline#179]
## Description

Provide scripts for users so that they can download internal and external data more easily. The files for these processes should live in scripts/ and pipeline/sources.

## External 
- [ ] VIAF
- [x] ORCID
- [x] ROR

**Comment by wjbmattingly** [comment]
I have modified how these scripts return data so that they return json dumps. Since we want to call them separately via bash scripts, this seemed like the easiest solution.

**Comment by kkdavis14** [comment]
most externals don't need specific download/parsers. the three that do have been completed in the v0.0.2 branch.

**Comment by kkdavis14** [comment]
Ideas for next steps on this:

- [ ] VIAF and ROR parsers are essentially this same, refactor to be more efficient, add the download URLs to their configs
- [ ] For LUX, we need the files not only to be downloaded but unzipped and placed in the correct directories in the pipeline machine so the loaders can grab them. Code connecting the download and stuffing in the right directories needs either refactored or created.
- [ ] For non-LUX/making this available as a service for others to use: once the above steps are completed, should be good to go.

**Comment by kkdavis14** [comment]
question: do we want all these sys.exits in the download/sources files, or print statements instead? when running programmatically, I think we don't want it to fail out if one source isn't working, but rather let us know which one and why.

**Comment by wjbmattingly** [comment]
@kkdavis14 no, we don't want all those sys.exits. We want these to be proper functions stored in the main pipeline in their respective paths. I was a bit worried about modifying those until your work was done as I didn't want to break anything for you in v.0.0.2. If you like, I can go ahead and do that work. It would be ready for you first thing Monday.

I think you are right about us not wanting it to fail out if one source isn't working. I will update the download.py script to align with that.

**Comment by kkdavis14** [comment]
I actually found the API the JS was pointing to by inspecting the code in the browser--much easier, now it's just a json parse. ROR is the same, both have been updated.

**Comment by kkdavis14** [comment]
> [@kkdavis14](https://github.com/kkdavis14) no, we don't want all those sys.exits. We want these to be proper functions stored in the main pipeline in their respective paths. I was a bit worried about modifying those until your work was done as I didn't want to break anything for you in v.0.0.2. If you like, I can go ahead and do that work. It would be ready for you first thing Monday.
> 
> I think you are right about us not wanting it to fail out if one source isn't working. I will update the download.py script to align with that.

sounds good to me. I think I am finished with that part of the work now. I can also take a brief pause on thinking about that part and focus on other v0.0.2 tickets while you work on that. Thanks!

**Comment by kkdavis14** [comment]
closing in favor of new repo.
